src/rprblender/engine/image_filter.py

```diff
@@ -112,28 +112,26 @@ def get_data(self):
         self.command_queue.synchronize()
         return self.output_image.get_data()
 
-    def setup_alpha_filter(self, input_buffer, alpha, output):
+    def setup_alpha_filter(self, alpha):
         """ Apply transparent background by setting output image alpha by alpha value """
         result = self.context.create_filter(rif.IMAGE_FILTER_USER_DEFINED)
 
         # redefine image alpha channel by alpha value
         code = \
             "int2 coord;" \
-            "GET_COORD_OR_RETURN(coord, GET_BUFFER_SIZE(outBuf));" \
-            "vec4 pixel = ReadPixelTyped(inputBuf, coord.x, coord.y);" \
+            "GET_COORD_OR_RETURN(coord, GET_BUFFER_SIZE(outputImage));" \
```

Comment:
Looks like using 3 quotes syntax for multiline string would suit better here:
code = """
.....
"""

Reply:
+1

Reply:
Yeah, that looks much better